web-frontend/package.json

```
@@ -17,7 +17,7 @@
     "prepare": "cd .. && husky install web-frontend/.husky"
   },
   "dependencies": {
-    "@massalabs/react-ui-kit": "^0.0.2-dev",
+    "@massalabs/react-ui-kit": "^0.0.4-dev.20230510134734",
```

please run npm i after applying this change.

```suggestion
    "@massalabs/react-ui-kit": "^0.0.4-dev",
```